src/BaGet.Core/Mirror/MirrorNuGetClientAdapter.cs

```diff
@@ -0,0 +1,8 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BaGet.Core.Mirror
+{
```

Comment:
Looks like this file should be deleted :)

Reply:
Done!